### PR TITLE
Proxy conf ii mnrva 605

### PR DESCRIPTION
**Pull request by FarajiA**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-605
Blocking: https://jira.rax.io/browse/MNRVA-603

 ## Description:
This is an update that will completely remove the `localenv` folder, and proxy all requests from the app to portal app. When setting up the app we will have the choice to decide whether to use the _staging_ or _prod_ version of identity & portal. 

 ## Testing:
`npm run start:dev`

 ## Screenshots:
proxyLogin

### CODE REVIEW

**Review by lovedeep92 — COMMENTED** [review]
Hey, I am getting **"An unhandled exception occurred: Script file scripts/portaldata.js does not exist."** when running `**npm run start:dev**`

File: angular.json [hunk 1]
```diff
@@ -90,20 +90,21 @@
                 }
               ],
               "scripts": [
-                "localenv/scripts/portaldata.js",
-                "localenv/scripts/pilotdata.js"
+                "scripts/portaldata.js",
```
Comment (pras7430) [inline angular.json:93]: hey, do we have a scripts folder?

**Review by pras7430 — COMMENTED** [review]
"npm run start:dev" script showing me below error. 

An unhandled exception occurred: Script file scripts/portaldata.js does not exist.

**Review by pras7430 — CHANGES_REQUESTED** [review]
fine!

File: angular.json (hunk 1 above)
Comment (FarajiA) [reply angular.json:93]: This is autogenerated

**Review by pras7430 — APPROVED** [review]

**Pull request merged**